prediction/src/algorithms/identify/helpers.py

```diff
@@ -0,0 +1,174 @@
+    interpolation = cv2.INTER_LINEAR
+    res = cv2.resize(images_zyx, dsize=(target_shape[1], target_shape[0]),
+                     interpolation=interpolation)  # opencv assumes y, x, channels umpy array, so y = z pfff
+
+    res = res.swapaxes(0, 2)
+    res = res.swapaxes(0, 1)
+
+    # cv2 can handle max 512 channels..
+    if res.shape[2] > 512:
+        res = res.swapaxes(0, 2)
+        res1 = res[:256]
+        res2 = res[256:]
+        res1 = res1.swapaxes(0, 2)
+        res2 = res2.swapaxes(0, 2)
+        res1 = cv2.resize(res1, dsize=(target_shape[2], target_shape[1]), interpolation=interpolation)
+        res2 = cv2.resize(res2, dsize=(target_shape[2], target_shape[1]), interpolation=interpolation)
+        res1 = res1.swapaxes(0, 2)
+        res2 = res2.swapaxes(0, 2)
+        res = numpy.vstack([res1, res2])
+        res = res.swapaxes(0, 2)
+    else:
+        res = cv2.resize(res, dsize=(target_shape[2], target_shape[1]), interpolation=interpolation)
+
+    res = res.swapaxes(0, 2)
+    res = res.swapaxes(2, 1)
+    return res
+
+
+def get_segmented_lungs(im):
+    # Step 1: Convert into a binary image.
+    binary = im < -400
+    # Step 2: Remove the blobs connected to the border of the image.
+    cleared = clear_border(binary)
+    # Step 3: Label the image.
+    label_image = label(cleared)
+    # Step 4: Keep the labels with 2 largest areas.
+    areas = [r.area for r in regionprops(label_image)]
+    areas.sort()
+    if len(areas) > 2:
+        for region in regionprops(label_image):
+            if region.area < areas[-2]:
+                for coordinates in region.coords:
+                    label_image[coordinates[0], coordinates[1]] = 0
+    binary = label_image > 0
+    # Step 5: Erosion operation with a disk of radius 2.
+    # This operation is seperate the lung nodules attached to the blood vessels.
+    selem = disk(2)
+    binary = binary_erosion(binary, selem)
+    # Step 6: Closure operation with a disk of radius 10.
+    # This operation is to keep nodules attached to the lung wall.
+    selem = disk(10)  # CHANGE BACK TO 10
+    binary = binary_closing(binary, selem)
+    # Step 7: Fill in the small holes inside the binary mask of lungs.
+    edges = roberts(binary)
+    binary = scipy.ndimage.binary_fill_holes(edges)
+    # Step 8: Superimpose the binary mask on the input image.
+    get_high_vals = binary == 0
+    im[get_high_vals] = -2000
+    return im, binary
+
+
+def load_patient(src_dir):
+    slices = []
+    for s in os.listdir(src_dir):
+        try:
+            slices.append(dicom.read_file(src_dir + '/' + s))
+        except Exception as e:
+            logging.error("{} no valid DICOM: {}".format(s, e))
+    slices.sort(key=lambda x: int(x.InstanceNumber))
+    try:
+        slice_thickness = numpy.abs(slices[0].ImagePositionPatient[2] - slices[1].ImagePositionPatient[2])
+    except Exception as e:
+        slice_thickness = numpy.abs(slices[0].SliceLocation - slices[1].SliceLocation)
+
```

Comment:
I think it'd be better to catch the specific exceptions in these cases. Using bare exceptions is usually code smell unless you plan to log and then reraise it.